sdk/storage/storage-blob/README.md

```diff
@@ -1,14 +1,17 @@
-# Azure Storage SDK V10 for JavaScript - Blob
+# Azure Storage client library for JavaScript - Blob
 
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob storage is optimized for storing massive amounts of unstructured data. Unstructured data is data that does not adhere to a particular data model or definition, such as text or binary data.
+
+This project provides a client library in JavaScript that makes it easy to consume Microsoft Azure Blob Storage service.
```

Comment:
`a client library` -> `the client library` ?

Reply:
I like `a` better.  `the` seems indicating the only one?

---
In reply to: [295981015](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r295981015) [](ancestors = 295981015)

Reply:
But, it is `the` only client library for JavaScript for storage-blob though, right ?

Reply:
there are older versions with different package names... maybe `a modern client library`? the thing is `modern` often got outdated

Reply:
We have legacy `azure-storage` npm package, which we called it as legacy storage JS sdk.

---
In reply to: [297371065](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r297371065) [](ancestors = 297371065)

Reply:
I will keep `a`

---
In reply to: [297673512](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r297673512) [](ancestors = 297673512,297371065)